conference/accounts.py

```diff
@@ -0,0 +1,62 @@
+# for some reason just doing from django.contrib.auth doesn't cut it for views
+from django.contrib.auth import (
+    forms as auth_forms,
+    views as auth_views,
+    authenticate,
+)
+from django.conf.urls import url
+from django import forms
+
+
+LOGIN_TEMPLATE = "ep19/bs/accounts/login.html"
+
+PRIVACY_POLICY_CHECKBOX = (
+    "I consent to the use of my data subject to the "
+    "<a href='/privacy/'>EuroPython data privacy policy</a>"
+)
+
+PRIVACY_POLICY_ERROR = (
+    "You need to consent to use of your data before we can continue"
+)
+
+
+class LoginForm(auth_forms.AuthenticationForm):
+    email = forms.EmailField()
+    password = forms.CharField(widget=forms.widgets.PasswordInput)
+    i_accept_privacy_policy = forms.BooleanField(label=PRIVACY_POLICY_CHECKBOX)
```

Comment:
yes, it's the same as currently/last year.
I think it's mostly to make sure that people can get familiar with privacy policy if they reuse the account from previous years, just in case something changed.